nbs/examples/Introduction.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>I think its way clearer for a reader to use an actual time series data set. Iy you want to use this syhentic data set I would motivate it. Something like:</p><p>We are going to creat a synthetic data set to illustrate the idea. Therefore the data set will have the following properties. </p><p>Regardles, I would highly recommend using datestamps in the <code>ds</code>  column. If you are using ints in the datestamp column please motivate or explain. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Added a clearer description of the synthetic data linking it to the vectors and matrices introduced.</p><p>Changed <code>ds</code> to date range. </p>